ProteoformSuiteInternal/ProteoformRelation.cs

```diff
@@ -94,10 +155,20 @@ public List<ProteoformRelation> find_nearby_relations(List<ProteoformRelation> a
                 nearby_relations = all_relations.Where(relation =>
                     relation.group_adjusted_deltaM >= lower_limit_of_peak_width && relation.group_adjusted_deltaM <= upper_limit_of_peak_width).ToList();
                 nearby_deltaM = nearby_relations.Select(r => r.delta_mass).Average();
+           // if(i==0) { this.running_sum = nearby_relations.Count(); }  //if running sum = only for in range decimals. 
             }
             return nearby_relations;
         }
 
+        public void calculate_running_sum(List<ProteoformRelation> relations)
+        {
+            double nearby_deltaM = this.group_adjusted_deltaM;
```

Comment:
I'm not sure this is the value you want. Running sum used to be the unadjusted group count, if I remember correctly. Here, you are using the adjusted delta mass after double-averaging.